Makefile

```diff
@@ -491,7 +491,7 @@ sensor-integration-test: build-prep test-prep
 	set -eo pipefail ; \
 	for package in $(shell git ls-files ./sensor/tests | grep '_test.go' | xargs -n 1 dirname | uniq | sort | sed -e 's/sensor\/tests\///'); do \
 		CGO_ENABLED=1 GODEBUG=cgocheck=2 MUTEX_WATCHDOG_TIMEOUT_SECS=30 GOTAGS=$(GOTAGS),test scripts/go-test.sh -p 4 -race -cover -coverprofile test-output/coverage.out -v ./sensor/tests/$$package \
-		| tee test-output/$$(echo $$package | sed -e 's/\//\_/').integration.log; \
+		| tee $(GO_TEST_OUTPUT_PATH); \
```

Comment:
I'm not sure if there was a reason to have the logs of each test in separate files. I'm ok with having them in one file though.